### DISCUSSION TITLE
What is the currency of 🇲🇽?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#825] [Galaxy Trivia] [answered]
**Bonus:** What other country outside of 🌎 uses this currency?

**Comment by peterbaccaro** [comment]
The currency of Mexico is the **Mexican Peso** (MXN).

Other countries that use pesos are:
- Argentina
- Chile
- Colombia
- Cuba
- Dominican Republic
- Philippines
- Uruguay

One fictional country that uses the currency known as the "Peso" is **San Theodoros**, a fictional South American country created by Belgian cartoonist Hergé in the comic series "The Adventures of Tintin." San Theodoros is featured in the Tintin adventure called "The Broken Ear." In this story, the currency used in San Theodoros is the "San Theodoran Peso."

**Comment by SANKARAMDAS** [comment]
🇲🇽 Mexican Peso (MXN)
The ISO 4217 number for this currency is 484.

There are currently eight countries that use the peso, almost all of which are in the Americas:
1 | Argentina | Argentine peso | ARS
2 | Chile | Chilean peso | CLP
3 | Colombia | Colombian peso | COP
4 | Cuba | Cuban peso; Cuban convertible peso | CUP; CUC
5 | Dominican Republic | Dominican peso | DOP
6 | Mexico | Mexican peso | MXN
7 | Philippines | Philippine peso | PHP
8 | Uruguay | Uruguayan peso | UYU

There are also 14 countries, including Spain, which previously used the peso.
https://www.worldatlas.com/articles/which-countries-use-the-peso-as-a-unit-of-currency.html

San Theodoros is a Latin American (fictional)country in The Adventures of Tintin also uses the "Peso" as a currency.
https://tintin.fandom.com/wiki/San_Theodoros

**Comment by ParanoidUser** [comment]
So what country outside of 🌎 still uses the peso?

**Reply by SANKARAMDAS** [reply]
San Theodoros is a Latin American (fictional)country in The Adventures of Tintin also uses the "Peso" as a currency.
https://tintin.fandom.com/wiki/San_Theodoros

Might be the E.T's planet it has... 🤣😂

**Reply by peterbaccaro** [reply]
In my original reply I mentioned San Theodoros however that fictional country would still exist on Earth.

I have no idea which country outside of earth would use the peso.  Any other clues?

**Reply by ParanoidUser** [reply]
A closer look at the emoji may reveal the content to be excluded. 🙃

**Reply by SANKARAMDAS** [reply]
![tintin](https://github.com/ParanoidUser/yolo/assets/31897843/0055113a-b974-4738-98e3-48ce479851e3)

![i284289739507193934 _szw1280h1280__1](https://github.com/ParanoidUser/yolo/assets/31897843/69a52201-7d5e-4ed1-8f43-400d45e4e2ef)

**Comment by anjutares** [comment]
The currency of 🇲🇽 Mexico is the Mexican Peso (MXN).

In several Spanish-speaking countries, the currency is called "peso." These countries are:
    Mexico (🇲🇽)
    Argentina (🇦🇷)
    Colombia (🇨🇴)
    Chile (🇨🇱)
    Uruguay (🇺🇾)
    Dominican Republic (🇩🇴)
    Cuba (🇨🇺)
    Philippines (🇵🇭)

**Comment by Carol42** [comment, marked as answer]
The official currency of Mexico 🇲🇽 is the Mexican Peso (Peso Mexicano), denoted by the symbol $ and currency code MXN. While several other countries also use Peso as their official currency, there is only one country outside the Americas 🌎 that adopts it. The Philippines 🇵🇭, located in Asia, utilizes the Philippine Peso (symbol: ₱; code: PHP)

**Reply by ParanoidUser** [reply]
Bingo! 👏
